packages/client/wallets/smart-wallet/src/api/CrossmintWalletService.ts

```diff
@@ -18,35 +17,18 @@ export class CrossmintWalletService extends BaseCrossmintService {
         );
     }
 
-    async getSmartWalletConfig(user: UserParams, chain: EVMBlockchainIncludingTestnet) {
+    async getSmartWalletConfig(
+        user: UserParams,
+        chain: EVMBlockchainIncludingTestnet
+    ): Promise<{ kernelVersion: string; entryPointVersion: string; signers: { signerData: SignerData }[] }> {
         return this.fetchCrossmintAPI(
-            `sdk/smart-wallet/versions?chain=${chain}`,
+            `sdk/smart-wallet/config?chain=${chain}`,
```

Comment:
u guys should really version your crossmint-main routes if they are not already, i think

Reply:
Agreed, we'll do so before launching

Reply:
done